src/views/Simulation/utils/MinimapContainer.js

```diff
@@ -18,24 +18,22 @@ export class MinimapContainer extends Container {
   }
 
   renderElements() {
-    this.removeChildren();
+    this.removeChildren().forEach((child) => child.destroy({ children: true }));
     this.miniSceneContainer = this.forgeContainer(this.sceneContainerRef.current);
+    const stage = this.minimapAppRef.current.stage;
```

Comment:
[nitpick] The `stage` const is declared too soon, it's not used before l.31 :smile: 
Keeping the scope of variables as short as possible helps with readability and bug prevention